coordinator/testing-services/src/main/java/io/stargate/testing/metrics/FixedClientInfoTagProvider.java

```diff
@@ -28,9 +31,21 @@ public class FixedClientInfoTagProvider implements ClientInfoMetricsTagProvider
 
   public static final String TAG_KEY = "clientInfo";
   public static final String TAG_VALUE = "fixed";
+  public static final String TAG_KEY_DRIVER_NAME = "driverName";
+  public static final String TAG_VALUE_DRIVER_NAME = "java";
```

Comment:
Since this is used for testing, maybe use a name that indicates it's for testing? Like `java-test-client` or something.